lib/devise-verifiable.rb

```diff
@@ -17,4 +17,4 @@ module Controller
 
 Devise.add_module :verifiable, model: 'devise/models/verifiable',
                                controller: :verifications,
-                               route: { verification: [:new, :create] }
+                               route: { verification: [nil, :new, :create] }
```

Comment:
Just a note here. This might seem a little strange, but this is what Devise does to enable the default URL helper in forms.

Check it out here: https://github.com/plataformatec/devise/blob/master/lib/devise/modules.rb#L6